src/components/Charts/utils.ts

```diff
@@ -250,3 +261,25 @@ export const getZoneByValue = (value: number, zones: Zones) => {
     ? zones[Level.HIGH]
     : zones[Level.MEDIUM];
 };
+
+export const computeTickPositions = (
+  minY: number,
+  maxY: number,
+  zones: Zones,
+) => {
+  const maxZones = zones[Level.MEDIUM].upperLimit;
+  const maxTick = maxY < maxZones ? 1.5 * maxZones : maxY;
+  return [
+    minY,
+    zones[Level.LOW].upperLimit,
+    zones[Level.MEDIUM].upperLimit,
+    maxTick,
+  ];
+};
+
+export const getAxisLimits = (minY: number, maxY: number, zones: Zones) => {
+  const tickPositions = computeTickPositions(minY, maxY, zones);
+  const minTickPosition = _.min(tickPositions) || minY;
+  const maxTickPosition = _.max(tickPositions) || maxY;
+  return roundAxisLimits(minTickPosition, maxTickPosition);
+};
```

Comment:
did we not need these for the rt graph?

Reply:
`computeTickPositions` was in the chart file, I moved to `utils` so I can reuse it for other charts.

Reply:
(also `getAxisLimits`)